OCKSample/Main/Care/CareView.swift

```diff
@@ -26,15 +32,22 @@ struct CareView: UIViewControllerRepresentable {
 
     func updateUIViewController(_ uiViewController: UIViewControllerType,
                                 context: Context) {
-        guard let navigationController = uiViewController as? UINavigationController else {
-            Logger.feed.error("View should have been a UINavigationController")
+        guard let navigationController = uiViewController as? UINavigationController,
+              let careViewController = navigationController.viewControllers.first as? CareViewController else {
+            Logger.feed.error("CareView should have been a UINavigationController")
+            return
+        }
+        guard careViewController.store !== appDelegate.storeCoordinator else {
+            // No need to replace view
+            // careViewController.events = events
```

Comment:
Commented out this line to prevent app crashing in Xcode 14.3 due to SwiftUI card and UIKit cards streaming in the same view but called on different queues, resulting in: `error: NULL _cd_rawData but the object is not being turned into a fault CoreData: error: NULL _cd_rawData but the object is not being turned into a fault`